### ISSUE TITLE
False positive on `Layout/FirstArgumentIndentation` cop

### ISSUE THREAD

**Issue by ydakuka** [rubocop/rubocop#12295]
## Actual behavior

I have:
```ruby
class UserPresenter
  PREFIXES.each do |prefix|
    define_method("#{prefix}_user_ids") do
      # ...

      instance_variable_set(
        "@#{prefix}_user_ids",
        User.where(
              'limit >= ? and limit < ?',
              LIMITS[prefix][:lowest_value], LIMITS[prefix][:highest_value]
            )
            .where(id: 1)
            .ids,
      )

      # ...
    end
  end
end
```

I run rubocop with `-A` option and I got:
```ruby
class UserPresenter
  PREFIXES.each do |prefix|
    define_method("#{prefix}_user_ids") do
      # ...

      instance_variable_set(
        "@#{prefix}_user_ids",
        User.where(
          'limit >= ? and limit < ?',
          LIMITS[prefix][:lowest_value], LIMITS[prefix][:highest_value]
        )
            .where(id: 1)
            .ids,
      )

      # ...
    end
  end
end
```

## Rubocop

```bash
ydakuka@yauhenid:~/Work/main_app$ bin/rails_docker rubocop -V
1.57.1 (using Parser 3.2.2.4, rubocop-ast 1.29.0, running on ruby 2.7.8) [x86_64-linux]
  - rubocop-capybara 2.19.0
  - rubocop-factory_bot 2.24.0
  - rubocop-performance 1.19.1
  - rubocop-rails 2.21.1
  - rubocop-rake 0.6.0
  - rubocop-rspec 2.24.1
  - rubocop-thread_safety 0.5.1
```

**Comment by ydakuka** [comment]
And second example:
```ruby
# before

class UserPresenter
  define_method("profiles_per_user_#{scope}") do
    # ...

    instance_variable_set("
      @profiles_per_user_#{scope}",
      User.was_healthy
          .where(id: 1)
    )

    # ...
  end
end
```
```ruby
# after

class UserPresenter
  define_method("profiles_per_user_#{scope}") do
    # ...

    instance_variable_set("
      @profiles_per_user_#{scope}",
                          User.was_healthy
                              .where(id: 1))

    # ...
  end
end
```
